lib/bolt_ext/server.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'bolt'
+require 'json'
+
+set :server, :puma
+
+post '/ssh/run_task' do
+  body = JSON.parse(request.body.read)
+  target = [Bolt::Target.new(body['target']['hostname'], body['target']['options'])]
+  task = Bolt::Task.new(body['task']['name'], body['task']['file'], body['task']['metadata'])
+  parameters = body['parameters']
+
+  @analytics = Bolt::Analytics.build_client
```

Comment:
Should we be using the analytics client? It seems like we should let something upstream handle analytics using the analytics service.

Reply:
Sure, and it could be an option passed on startup? Do we want to collect bolt analytics for these jobs? If so, do we need to pass the analytics client to the executor to get analytics for the jobs, or can the client handle analytics on it's own?